python/test/hive/util/test_hive_ddl_extractor.py

```diff
@@ -0,0 +1,71 @@
+"""
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+"""
+
+import dataproc_templates.util.template_constants as constants
+import mock
+import pyspark
+from dataproc_templates.hive.util.hive_ddl_extractor import \
+    HiveDDLExtractorTemplate
+
+
+class TestHiveDDLExtractorTemplate:
+    """
+    Test suite for HiveDDLExtractorTemplate
+    """
+
+    def test_parse_args(self):
+        """Tests HiveDDLExtractorTemplate.parse_args()"""
+
+        hive_ddl_extractor_template = HiveDDLExtractorTemplate()
+        parsed_args = hive_ddl_extractor_template.parse_args(
+            ["--hive.ddl.extractor.input.database=database",
+             "--hive.ddl.extractor.output.dataset=dataset",
+             "--hive.ddl.extractor.output.table=table",
+             "--hive.ddl.extractor.output.bucket=bucket"])
```

Comment:
You would have to ajust the unit test after changes the gcs location and bq temp bucket parameters